contracts/factory/src/tests/config.rs

```diff
@@ -107,6 +113,92 @@ fn factory_successfully_inits_lp() {
     );
 }
 
+#[test]
+fn factory_successfully_inits_stable_pool() {
+    let env = Env::default();
+    env.mock_all_auths();
+    env.budget().reset_unlimited();
+
+    let admin = Address::generate(&env);
+    let token1_admin = Address::generate(&env);
+    let token2_admin = Address::generate(&env);
+    let user = Address::generate(&env);
+
+    let mut token1 = install_and_deploy_token_contract(
+        &env,
+        &token1_admin.clone(),
+        &7,
+        &String::from_str(&env, "Phoenix"),
+        &String::from_str(&env, "PHO"),
```

Comment:
nit: Let's make it more legit. ;)

```suggestion
        &String::from_str(&env, "EURC"),
        &String::from_str(&env, "EURC"),
```